test/WebSites/BasicWebSite/Views/RemoteAttribute_Home/Create.cshtml

```diff
@@ -0,0 +1,75 @@
+﻿@model BasicWebSite.Models.RemoteAttributeUser
```

Comment:
Odd this and other views show up as additions. Is the old `ValidationWebSite` completely clean?